worlds/x2wotc/mods/lwotc/Rules.py

```diff
@@ -0,0 +1,48 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from worlds.x2wotc import X2WOTCWorld
+from ...Rules import RuleManager
+
+from BaseClasses import MultiWorld, CollectionState
+from worlds.generic.Rules import set_rule, add_rule
+
+
+from ...LocationData import X2WOTCLocationData
```

Comment:
There's a bunch of unused imports, but this is also clearly still WIP.
```suggestion
from BaseClasses import CollectionState
```